sdk/monitor/azure-monitor-query/azure/monitor/query/_models.py

```diff
@@ -69,26 +68,22 @@ class LogsQueryResults(object):
 
     :keyword tables: The list of tables, columns and rows.
     :paramtype tables: list[~azure.monitor.query.LogsQueryResultTable]
-    :keyword errors:
-    :paramtype errors: ~azure.monitor.query.LogsErrorDetails
+    :keyword statistics: Any object.
+    :paramtype statistics: object
+    :keyword render: Any object.
+    :paramtype render: object
     """
     def __init__(self, **kwargs):
         # type: (Any) -> None
         self.tables = kwargs.get("tables", None)
-        self.errors = kwargs.get("errors", None)
+        self.statistics = kwargs.pop("statistics", None)
+        self.render = kwargs.pop("render", None)
```

Comment:
should the last two also be `get` for consistency? how does it affect what is being passed in? or does it not matter?